tests/virtual_machines_test.go

```diff
@@ -515,27 +515,46 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 
 		Context("When the leader is changed", func() {
```

Comment:
"When a VM is created at opt-in namespace and leadership is changed"

Reply:
I think all contexts except the 1 (the opted out) is opt in by default.
Should I change it anyway?

Reply:
Nah all good.